src/lib/product_reviews/visualizer/index.js

```diff
@@ -89,7 +109,16 @@ class Visualizer extends Component {
            "ratingValue": average,
            "reviewCount": numReviews
          },
-         "review": [schemaReviews],
+         "brand": {
+            "@type": "Brand",
+            "name": "iFixit",
```

Comment:
Will this be true for every product that uses this component?

Reply:
Yeah me neither, but if we don't have the data we should probably just not include it